packages/reassure-measure/src/measure-function.tsx

```diff
@@ -0,0 +1,41 @@
+import { config } from './config';
+import { RunResult, processRunResults } from './measure-helpers';
+import { showFlagsOuputIfNeeded, writeTestStats } from './output';
+import type { MeasureResults } from './types';
+
+export interface MeasureFunctionOptions {
+  runs?: number;
+  dropWorst?: number;
+}
+
+export async function measureFunction(fn: () => void, options?: MeasureFunctionOptions): Promise<MeasureResults> {
+  const stats = await measureFunctionInternal(fn, options);
+  await writeTestStats(stats);
+
+  return stats;
+}
+
+export function measureFunctionInternal(fn: () => void, options?: MeasureFunctionOptions): MeasureResults {
+  const runs = options?.runs ?? config.runs;
+  const dropWorst = options?.dropWorst ?? config.dropWorst;
+
+  showFlagsOuputIfNeeded();
+
+  const runResults: RunResult[] = [];
+  for (let i = 0; i < runs + dropWorst; i += 1) {
+    const timeStart = getCurrentTime();
+    fn();
+    const timeEnd = getCurrentTime();
+
+    global.gc?.();
+
+    const duration = Number((timeEnd - timeStart) / 1_000_000n);
+    runResults.push({ duration, count: 1 });
+  }
+
+  return processRunResults(runResults, dropWorst);
+}
+
+function getCurrentTime() {
+  return process.hrtime.bigint();
```

Comment:
How about using performance API from perf_hooks? https://nodejs.org/api/perf_hooks.html#performancenow

Reply:
I've thought about it and done some prototyping around. My conclusions are that in this very basic API it's not needed and would only complicate the API for the users. 

I am actively contemplating using Performance API in context of allowing multiple measurement per test. That would be a more complex feature for advanced users. It could also be used in both render and function measuring.

Reply:
We should be using `performance.now()` due to monotonous clock being gauranteed there.